util/buildRelease/gen_release

```diff
@@ -212,16 +212,13 @@ SystemOrDie("rm examples"); # remove examples symbolic link
 SystemOrDie("cp -r test/release/examples .");
 SystemOrDie("cd util && cp start_test ../examples/");
 
-print "[gen_release] Removing Makefiles that are not intended for release...\n";
-SystemOrDie("cd make/platform && rm Makefile.sunos_old");
+# print "[gen_release] Removing Makefiles that are not intended for release...\n";
 
-print "[gen_release] Removing compiler directories that are not intended for release...\n";
-SystemOrDie("cd compiler/include && rm -r sunos_old");
+# print "[gen_release] Removing compiler directories that are not intended for release...\n";
```

Comment:
I kept them as comments in case we want to add similar stuff here. But I don't know about the release process enough to gauge whether there's any value in it.